alarms/dojot.auth.PermissionError.xml

```diff
@@ -0,0 +1,12 @@
+<alarm-metamodel>
+       <domain>PermissionError</domain>
```

Comment:
What is this used for? And how is this different from `AuthorizationError`?

Reply:
I willl remove this one